src/app/components/home/post-list/post-list.component.html

```diff
@@ -1,19 +1,22 @@
 <!-- Page content start -->
 <div class="container">
     <div class="row">
+
         <!-- Blog entries start -->
         <div class="col-lg-8">
+            <div class="text-center mt-4 mb-4">
+                <h1 class="fw-bolder">Ultimele postari</h1>
+            </div>
             <div *ngIf="posts$ | async as posts" class="row">
                 <div class="row" *ngIf="posts.length > 0">
-                    <div *ngFor="let post of posts" class="col-lg-6">
+                    <div *ngFor="let post of posts">
                         <mat-card class="mb-4">
-                            <mat-card-title>{{ (post.title.length > 30) ? (post.title | slice:0:30) + ' ...' :
+                            <mat-card-title>{{ (post.title.length > 50) ? (post.title | slice:0:50) + '...' :
```

Comment:
You could create a PIPE in Angular for this type of formatting

Reply:
@MirceaDumitrescu thank you for your idea. I've created a custom pipe.